### ISSUE TITLE
For the matter of Spatial Resolution(7) in MD Fields

### ISSUE THREAD

**Issue by hyekyoung-lee** [OpenWIS/excel2wis#3]
Spatial Resolution(7) is declared as an optional in MD Fields.
However, no input information in Spatial Resolution(7) makes an error in saving after editing the metadata in admin portal ( in Metadata catalogue content > Edit > save )

We need to check if this is excel2wis problem or OpenWIS bug.

**Comment by rogers492** [comment]
@mclaudon-mto Are you the right person to look into this? If not, please assign to whoever it is. Thanks.

**Comment by benjaminsaclier** [comment]
I confirm that these fields are optionals. I did some test by editing a metadata which have some spatial resolution information and I removed it without any error after saving (see screenshot) .
![firefox_screenshot_2017-05-23t15-02-33 198z](https://cloud.githubusercontent.com/assets/12250260/26361017/f2de728c-3fd9-11e7-8e3b-0e174c9c27a0.png)

**Comment by rogers492** [comment]
@hyekyoung-lee Does the response from @benjaminsaclier solve your problem?

**Comment by SungsooDO** [comment]
Yes, spatial resolution fields are defined as "optional" in the code so this issue can be closed.
